### DISCUSSION TITLE
Schema differences between Oracle and MsSQL script - indexes

### DISCUSSION THREAD

**Discussion by aszentmihalyi** [quartznet/quartznet#1071] [General] [answered]
Comparing the Oracle and the MSSQL script I've found that indexes are different in the 2 scripts.
Eg.:
- "idx_qrtz_j_req_recovery" and "idx_qrtz_j_grp"  indexes exist in Oracle script but not in MSSQL. 
- "create index idx_qrtz_t_j on qrtz_triggers(SCHED_NAME,JOB_NAME,JOB_GROUP);" in Oracle script is similar but not the same as "CREATE INDEX [IDX_QRTZ_T_G_J]                 ON [dbo].[QRTZ_TRIGGERS](SCHED_NAME, JOB_GROUP, JOB_NAME);" in the MSSQL script.
- "create index idx_qrtz_t_jg on qrtz_triggers(SCHED_NAME,JOB_GROUP);" is missing from MSSQL script
- and so on..

Of course it looks like everything works well in spite of this differences, but are there any specific reason for this?

**Comment by lahma** [comment, marked as answer]
The index schema changes are quite recent thing. I don't like to struggle with Oracle so these haven't been ported,  not even sure if they make a difference there. 

SQL Server version was optimized to minimize overhead while keeping the covering.
